kernelci/api/helper.py

```diff
@@ -111,6 +112,14 @@ def create_job_node(self, job_config, input_node):
             'artifacts': input_node['artifacts'],
             'revision': input_node['revision'],
         }
+        # This information is highly useful, it might be:
+        # lab-name, device-name or kubernetes, cluster name
```

Comment:
I think there is a redundant `,` here.
```suggestion
        # lab-name, device-name or kubernetes cluster name
```

Reply:
No, its supposed to be like this, two parameters, in case of tests its lab name and device name, in case of kubernetes build its another two parameters

Reply:
Can you reformat the comment to make it a bit clearer?

Reply:
done